src/modules/project/pages/projectCreate/views/ProjectCreateCompletionPage.tsx

```diff
@@ -4,30 +4,35 @@ import { useTranslation } from 'react-i18next'
 import { BiRocket } from 'react-icons/bi'
 import { Navigate, useNavigate, useParams } from 'react-router-dom'
 
-import TitleWithProgressBar from '../../components/molecules/TitleWithProgressBar'
-import Loader from '../../components/ui/Loader'
-import { getPath } from '../../constants'
-import { useAuthContext } from '../../context'
-import { useModal } from '../../hooks'
-import { CreateWalletInput, ProjectFragment, useProjectByNameOrIdQuery, useProjectPublishMutation } from '../../types'
-import { toInt, useNotification } from '../../utils'
-import { ProjectCreateCompleted } from './components/ProjectCreateCompleted'
-import { ProjectCreateLayout } from './components/ProjectCreateLayout'
-import { ProjectLaunchConfirmModal } from './components/ProjectLaunchConfirmModal'
-
-interface ProjectCreateCompletionProps {
+import TitleWithProgressBar from '../../../../../components/molecules/TitleWithProgressBar'
```

Comment:
This is where things start getting very nested and we might want to consider using path aliases to have something like `@/components/...` (cf: https://blog.logrocket.com/using-path-aliases-cleaner-react-typescript-imports/)

Reply:
thanks stelios, that was a good read. It will definitely make things cleaner, and moving things from one file to another will be simpler too.